src/components/icon/icon.style.js

```diff
@@ -123,6 +123,7 @@ const StyledIcon = styled.span`
       position: relative;
       color: ${finalColor};
       background-color: ${bgColor};
+      vertical-align: middle;
```

Comment:
Just noticed the Chromatic changes caused by that
https://www.chromatic.com/pullrequest?appId=5ecf782fe724630022d27d7d&number=3443&view=changes

Is that ok?